api/src/main/java/org/pmiops/workbench/auth/ServiceAccounts.java

```diff
@@ -1,52 +1,26 @@
 package org.pmiops.workbench.auth;
 
-import static com.google.api.client.googleapis.util.Utils.getDefaultJsonFactory;
-
-import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
-import com.google.api.client.http.HttpTransport;
-import com.google.appengine.api.appidentity.AppIdentityService;
-import com.google.appengine.api.appidentity.AppIdentityServiceFactory;
+import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
 import java.util.List;
-import org.pmiops.workbench.config.WorkbenchEnvironment;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
 
 /**
  * Handles functionality related to loading service account credentials and generating derived /
  * impersonated credentials.
  */
-@Component
 public class ServiceAccounts {
```

Comment:
I was very pro-utility class until I realized just how much Spring and Mockito aren't excited about working with them. If there are unit tests you want to write that depend on any of these functions, you can't inject a mock.